src/gmpy2.c

```diff
@@ -780,6 +781,7 @@ static PyMethodDef Pygmpy_methods [] =
     { "mul", GMPy_Context_Mul, METH_VARARGS, GMPy_doc_function_mul },
     { "multi_fac", GMPy_MPZ_Function_MultiFac, METH_VARARGS, GMPy_doc_mpz_function_multi_fac },
     { "next_prime", GMPy_MPZ_Function_NextPrime, METH_O, GMPy_doc_mpz_function_next_prime },
+    { "prev_prime", GMPy_MPZ_Function_PrevPrime, METH_O, GMPy_doc_mpz_function_prev_prime },
```

Comment:
This you should do also conditionally.  Ditto in [src/gmpy2_mpz_misc.h](https://github.com/aleaxit/gmpy/pull/284/files#diff-98ae8bc1d201bda720b449caed6fa3091db148f52c914b5a959604b9e91fabf0)